### PR TITLE
#161918412 [create nodejs project structure]

### PR DESCRIPTION
**Pull request by barakimam**
#### What does this PR do?
- add project readme with basic dev environment setup instructions
- add basic nodejs project structure
- add sample /slack endpoint

#### Description of Task to be completed?
- create a nodejs project structure
- start a basic nodejs app with an express.js server
- implement the `/api/v1/slack` endpoint

#### How should this be manually tested?
- follow the readme and try to start the app, enter `http://localhost:5000/api/v1/slack` in the browser. You should get a welcome message.

#### Any background context you want to provide?
The docker bits are not done yet.

#### Questions:
Contact barak on Slack.

### CODE REVIEW

**Comment by reiosantos** [comment]
I thought we were going to use ES6, but I am seeing ES5 being used. Could there be a reason for this?

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = {
```
[the remark this answers is not in the document]
Comment (PAbishai) [reply .sequelizerc]: Thanks for the explanation @tomiadebanjo and @jamesenejo

[the remark this answers is not in the document]
Comment (barakimam) [reply .sequelizerc]: @PAbishai, the lines of code above is merely some configuration details of a command line tool for database management. The tool is sequelizer and it still only support ES5.
Every bit of code that is core to the application will and is written in ES6+.

**Comment by barakimam** [comment]
> I thought we were going to use ES6, but I am seeing ES5 being used. Could there be a reason for this?

We will be using ES6+, the code-chunk above wasn't really code but sequelizer configuation details.

**Comment by DerKip** [comment]
LGTM

**Pull request merged**